scripts/dialog_xls2xml.py

```diff
@@ -34,23 +34,23 @@ def saveDialogDataToFileSystem(dialogData, handler, config):
             xmlData = handler.convertDialogData(dialogData, domains[domain])
             dialogFile.write(handler.printXml(xmlData))
 
-    if hasattr(config, 'common_generated_intents') and not os.path.exists(getattr(config, 'common_generated_intents')[0]):
-        os.makedirs(getattr(config, 'common_generated_intents')[0])
-        print('Created new directory ' + getattr(config, 'common_generated_intents')[0])
+    if hasattr(config, 'common_generated_intents') and not os.path.exists(getattr(config, 'common_generated_intents')):
+        os.makedirs(getattr(config, 'common_generated_intents'))
+        print('Created new directory ' + getattr(config, 'common_generated_intents'))
 
     for intent, intentData in dialogData.getAllIntents().iteritems():
         if len(intentData.getIntentAlternatives()) > 0:
-            with open(os.path.join(getattr(config, 'common_generated_intents')[0], intent.decode('utf8') + '.csv'), 'w') as intentFile:
+            with open(os.path.join(getattr(config, 'common_generated_intents'), intent.decode('utf8') + '.csv'), 'w') as intentFile:
                 for alternative in intentData.getIntentAlternatives():
                     intentFile.write(alternative.encode('utf8') + '\n')
 
-    if hasattr(config, 'common_generated_entities') and not os.path.exists(getattr(config, 'common_generated_entities')[0]):
-        os.makedirs(getattr(config, 'common_generated_entities')[0])
-        print('Created new directory ' + getattr(config, 'common_generated_entities')[0])
+    if hasattr(config, 'common_generated_entities') and not os.path.exists(getattr(config, 'common_generated_entities')):
+        os.makedirs(getattr(config, 'common_generated_entities'))
+        print('Created new directory ' + getattr(config, 'common_generated_entities'))
 
     entities = dialogData.getAllEntities()
     for entity in entities:
-        with open(os.path.join(getattr(config, 'common_generated_entities')[0], entity.encode('ascii', 'ignore') + '.csv'), 'w') as entityFile:
+        with open(os.path.join(getattr(config, 'common_generated_entities'), entity.encode('ascii', 'ignore') + '.csv'), 'w') as entityFile:
```

Comment:
More conversion of generated fields to non-arrays

Reply:
Removed.